packages/block-library/src/video/edit.js

```diff
@@ -150,7 +152,7 @@ class VideoEdit extends Component {
 		if ( editing ) {
 			return (
 				<MediaPlaceholder
-					icon="media-video"
+					icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
					icon={ <BlockIcon icon={ icon } /> }
```